package.json

```diff
@@ -71,7 +71,7 @@
     "@vitest/ui": "^3.1.1",
     "cross-env": "^7.0.3",
     "drizzle-kit": "^0.30.6",
-    "electron": "35.1.4",
+    "electron": "^35.7.5",
```

Comment:
let's leave this alone - we can upgrade electron separately

Reply:
Done